cli/commands/project_sync.go

```diff
@@ -39,6 +39,18 @@ func (cmd *ProjectSync) Commands() []cli.Command {
 		Usage:       "Start a unison sync on local project directory. Optionally provide a volume name.",
 		ArgsUsage:   "[optional volume name]",
 		Description: "Volume name will be discovered in the following order: argument to this command > outrigger project config > docker-compose file > current directory name",
+		Flags: []cli.Flag{
+			cli.IntFlag{
+				Name:   "initial-sync-timeout",
+				Value: 60,
+				Usage:  "Maximum amount of time in seconds to allow for detection of successful initial sync.",
+			},
+			cli.IntFlag{
+				Name:   "container-start-timeout",
+				Value: 10,
+				Usage:  "Maximum amount of time in seconds to wait for the unison container to start",
+			},
+		},
```

Comment:
Sans a desire to detect an issue as soon as possible the answer is likely no. I've actually not seen anyone suffer from waiting for the container to start timing out, it was added as a function of "since I'm in here . . . "